recipes/lyx/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "lyx" %}
+{% set version = "2.13.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.lip6.fr/pub/lyx/stable/2.3.x/lyx-2.3.6.1.tar.gz
+  sha256: 6d6f5458ebaac644cdfa35114d029e9ec57b4d930268d6d40bd9795d5c7e7929
+
+build:
+  skip: true  # [not linux]
+  number: 0
+  rpaths:
+    - lib/
```

Comment:
This shouldn't be needed
```suggestion
```

Reply:
Thanks, I guess I learned bad habits from conda_r_skeleton_helper...